docs/js.md

```diff
@@ -0,0 +1,162 @@
+### [Primative: Null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null)
+
+* _symbol_: `null`
+* _pronunciation_: "null", "null value", "is null"
+* _examples_:
+```javascript
+null // how exciting
+```
+
+### [Primative: Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)
+
+*_symbol_: 'true' 'false'
+*_pronunciation_"boo-lee-an"
+*_examples_:
+```javascript
+50 < 100 == true
+50 > 100 == false
+
+('meh' != meh) == true
+```
+
+### [Primative: Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)
+
+*_symbol_: 'The numeric value of the object being created.'
+*_pronunciation_"number" "integer"
+*_examples_:
+```javascript
+1
+2
+3
+10
+```
+
+### [Primative: String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)
+
+*_symbol_: 'a sequence of characters'
```

Comment:
Well, a sequence of characters _surrounded_ by single (`'`) or double (`"`) quotes. Any _other_ way to create a `String` value?